fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/AggregationAverageMaker.java

```diff
@@ -35,6 +38,10 @@
  * takes aggregated data from a finer time grain (i.e. DefaultTimeGrain.DAY) and computes an average across a coarser
  * time grain (i.e. DefaultTimeGrain.WEEK). For example, given the total number of visitors to www.example.com for each
  * day of the week, we can compute the average number of daily visitors to example.com for the entire week.
+ * <p>
+ * A nested average requires the following columns, an inner query constant, and outer query sum of that constant,
```

Comment:
Should there be a colon after `following columns` instead of a comma?

Also this paragraph is extremely dense, and I couldn't keep my eyes from glazing as I looked at it. I would suggest something like:

1. An inner query constant `c` to sum, and the outer query sum of `c`: `cs`.
2. An inner query (post) aggregation `a` to sum, and an outer query sum of `a`: `as`
3. An outer query post aggregation that computes `as / cs`.